code/modules/crafting/reloading.dm

```diff
@@ -0,0 +1,298 @@
+// /datum/crafting_recipe
+//	var/name = "" //in-game display name
+//	var/reqs[] = list() //type paths of items consumed associated with how many are needed
+//	var/result //type path of item resulting from this craft
+//	var/tools[] = list() //type paths of items needed but not consumed
+//	var/time = 30 //time in deciseconds
+//	var/parts[] = list() //type paths of items that will be placed in the result
+//	var/category = CAT_NONE //where it shows up in the crafting UI
+//	var/subcategory = CAT_NONE
+//9mm 10mm 357. 44. 45. 5.56 7.62
+// in the future, if you want to add more calibers, you'll need to create a bullet, bullet recipie, die, die recipie, and bullet creation recipies.
+// to adjust scarcity levels, change the amount of bullets that you get with one metal sheet from five to a lower number
+// made by john oxford
+
+// Equipment
+
+/obj/item/reloading/die/reloadingpress
+	name = "reloading press"
+	desc = "An Oxford Armory� universal combination reloading press, complete with a primer synthesizer, automatic powder measuring, and a high tourqe pressing rod."
+	tool_behaviour = TOOL_RELOADING
+	attack_verb = list("clonked", "smashed", "chunked", "whacked")
+	icon = 'icons/obj/reloading.dmi'
+	icon_state = 'reloadingpress'
+
+// Dies
+
+/obj/item/reloading/die
+	name = "reloading die"
+	desc = "This probally shouldn't exist, quit admin abusing."
+	attack_verb = list("cracked", "chipped", "smashed", "clocked")
+	icon = 'icons/obj/reloading.dmi'
+
+/obj/item/reloading/die/ninemilimeter
+	name = "9mm reloading die"
+	desc = "A sandstone reloading die with 5 slots each carved into the shape of a 9mm bullet. You vaguely can make out the letters O and X on the bottom of the brick."
+	icon_state = 'diesmall'
+
+/obj/item/reloading/die/tenmilimeter
+	name = "10mm reloading die"
+	desc = "A sandstone reloading die with 5 slots each carved into the shape of a 9mm bullet. You vaguely can make out the letters O and X on the bottom of the brick."
```

Comment:
10mm in the shape of a 9mm bullet, nice. Wow, the 9mm bullet is a very versatile shape!

Reply:
i can't denote a difference of .1 mm in a 32x32 sprite

Reply:
desc = "... carved shape of 9mm bullet" for all of the dies